internal/mesh/internal/controllers/sidecarproxy/builder/destinations.go

```diff
@@ -71,7 +71,7 @@ func (b *Builder) buildDestination(
 			}
 		}
 	}
-
+	
```

Comment:
Nit: whitespace

Reply:
thank you. fixed in b9c43cfa5cc6fe3c6b368d8ef94c3ce1a72db341